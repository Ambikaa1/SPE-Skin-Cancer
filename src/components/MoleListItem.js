import React from "react";
import { View, Text, StyleSheet, Image } from "react-native";

const DisplayImages = ({ uri, name, comments, lastUpdated }) => {
    return(
        <>
            <Image
                style = {styles.image}
                source = {{ uri: uri }}
            />
            <View style = {styles.moleInfo}>
                <Text style = {styles.moleName}>{name}</Text>
                <Text style = {styles.moleDetails}>{comments}</Text>
                <Text style = {styles.moleDetails}>Last updated: {lastUpdated}</Text>
            </View>
        </>
    );
};

const styles = StyleSheet.create({
    image: {
        height: 200,
        width: 100
    },
    moleInfo: {
        marginLeft: 10,
    },
    moleName: {
        fontSize: 20,
        fontWeight: "bold",
        paddingTop: 10,
        paddingBottom: 10,
    },
    moleDetails: {
        paddingBottom: 10,
    },
});

export default DisplayImages;