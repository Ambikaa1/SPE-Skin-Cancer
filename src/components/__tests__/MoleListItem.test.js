import React from 'react';
import renderer from 'react-test-renderer';

import MoleListItem from "../MoleListItem";

describe('<MoleListItem />', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<MoleListItem />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
