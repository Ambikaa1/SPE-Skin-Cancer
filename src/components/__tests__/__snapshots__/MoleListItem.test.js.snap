// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MoleListItem /> renders correctly 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "#71A1D1",
      "borderBottomWidth": 5,
      "borderTopColor": "#71A1D1",
      "borderTopWidth": 5,
      "flexDirection": "row",
      "marginHorizontal": 10,
      "marginVertical": 5,
      "paddingVertical": 5,
    }
  }
>
  <Image
    source={
      Object {
        "uri": undefined,
      }
    }
    style={
      Object {
        "height": 200,
        "width": 100,
      }
    }
  />
  <View
    style={
      Object {
        "flex": 1,
        "marginLeft": 10,
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 20,
          "fontWeight": "bold",
          "paddingBottom": 10,
          "paddingTop": 10,
        }
      }
    />
    <Text
      style={
        Object {
          "paddingBottom": 10,
        }
      }
    />
    <Text
      style={
        Object {
          "paddingBottom": 10,
        }
      }
    >
      Last updated: 
    </Text>
  </View>
</View>
`;
